tox.ini

```diff
@@ -22,3 +22,13 @@ paths =
 # NB: (Pinned) pylint is installed as part of the lektor[test] extras
 commands =
     pylint {posargs:lektor setup.py tests}
+
+[flake8]
+max-line-length = 100
```

Comment:
Is this the current standard? If so, it's not very friendly accessibility-wise. It may cause poor developer experience for people who use large fonts and vertical panels with 2-3 files open side-by-side sometimes.

Reply:
I copied it from the existing [pylintrc](https://github.com/lektor/lektor/blob/14324ab33f51d488783efd8d162f199e3c784eff/pylintrc#L5).  (The `pylintrc` should probably be deleted, since I don't think we currently use pylint.)

The current code-base has python lines of up to 91 characters in length.
It seems that `black` wraps (by default?) to a line-length of 88.  The lines in our current code-base longer than that appear to be comments and strings (which `black` is apparently okay with, I guess.)

Our `.editorconfig` sets a line-length of 88.

What should we use for a line length?

Reply:
> The pylintrc should probably be deleted, since I don't think we currently use pylint.

We still do use pylint.

>It seems that black wraps (by default?) to a line-length of 88. 

Yes, indeed (see https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length for why it uses a value higher than PEP8). Let's just set the lowest value that doesn't throw up any errors (so 91 I guess) in this PR since that seems to be a rather unrelated topic. In general I'd be fine with either the black default of 88 or 79 or 80 but I also assume that the 10% difference won't be too stark accessibility-wise.

Reply:
> We still do use pylint.

Maybe this is a discussion for another thread, but why are we using both `pylint` and `flake8`?  Isn't one or the other sufficient?

Reply:
IMHO pylint would be enough since we're also using black so there's not much formatting stuff to complain about for flake8. It's still good to have config for both so that they work for contributors who have these linters installed as editor plugins or such.